src/Service.Tests/Unittests/SqlMetadataProviderUnitTests.cs

```diff
@@ -102,6 +102,7 @@ public async Task CheckExceptionForBadConnectionStringForMsSql(string connection
         /// <code>Check: </code> Making sure table prefix matches expected prefix.
         /// </summary>
         [DataTestMethod]
+        [TestCategory(TestCategory.MSSQL)]
```

Comment:
This test looks for `dab-config.MsSql.json` file but it is unavailable in the build pipeline. The logic for fetching the file cannot be modified to read from `dab-config.MsSql.reference.json` as other tests also share that logic. 
Hence, running this test as part of the MsSql integration test pipeline (The other tests that use the same helper method to look for `dab-config.MsSql.json` already run as part of the MsSql integration test pipeline)

Reply:
can we move the generation of `dab-config.MsSql.json` from the integration test pipeline to the build pipeline so that these unit tests stay as unit tests?